website/docs/user_security.md

```diff
@@ -0,0 +1,294 @@
+---
+id: user_security
+title: User Security
+sidebar_label: User Security
+---
+
+`Core-Lib` has classes that provide user security implementations that the user may tailor to their needs for authorization or authentication purposes.
+These classes function together, and the user must implement them in order for `User Security` to work as expected.
+
+Classes that handle `User Security`
+## UserSecurity
+`UserSecurity` is an abstract class that provides security functions to be implemented by user as per their needs. Must be extended in the class where we will implement the abstract methods.

```

Comment:
grammerly?

Reply:
tried the same sentence with Grammarly it was not showing any errors

Reply:
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/5166849/157797435-04c6a1c2-22b9-4218-909a-79003a46eae6.png">